data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/exception/LogstashGrammarException.java

```diff
@@ -0,0 +1,13 @@
+package org.opensearch.dataprepper.logstash.exception;
+
+/**
+ * Exception thrown when ANTLR fails to parse the Logstash configuration
+ *
+ * @since 1.2
+ */
+public class LogstashGrammarException extends LogstashParsingException {
```

Comment:
This exception is never used. Will it be used in the future?

Reply:
Yes. The intention was to use it in the future.